agent/bench-scripts/pbench-fio

```diff
@@ -598,13 +598,15 @@ function fio_run_job() {
 
 	if [ -n "${clients}" ]; then
 		for client in "${client_names[@]}"; do
+			local bash_c_cmd="${benchmark_bin} --server=,${client_ports[${client}]} > client-result.txt 2> client-result.err"
```

Comment:
This should do, you might consider escaping the `${benchmark_bin}` in case it contains extra characters, but it wasn't escaped before either...